Troupon.Checkout.Api/DependencyInjectionExtensions/AddOpenApiExtensions.cs

```diff
@@ -1,13 +1,17 @@
-﻿using System;
```

Comment:
@pedro-beslogic you probably get the last version of this file from "catalog". This has been extracted to Infra too. I will take care of it in my task